karma.conf.js

```diff
@@ -0,0 +1,75 @@
+// Karma configuration
+// Generated on Mon Jan 26 2015 16:51:27 GMT+0100 (CET)
+
+module.exports = function(config) {
+  "use strict";
+  config.set({
+
+    // base path that will be used to resolve all patterns (eg. files, exclude)
+    basePath: '',
+
+
+    // frameworks to use
+    // available frameworks: https://npmjs.org/browse/keyword/karma-adapter
+    frameworks: ['browserify', 'mocha', 'chai'],
+
+
+    // list of files / patterns to load in the browser
+    files: [
+      'src/js/app.js',
+      'src/test/*_test.js'
+    ],
+
+
+    // list of files to exclude
+    exclude: [
+      'src/**/main.js'
+    ],
+
+
+    // preprocess matching files before serving them to the browser
+    // available preprocessors: https://npmjs.org/browse/keyword/karma-preprocessor
+    preprocessors: {
+      'src/**/*.js': ['browserify'],
+      'test/**/*.js': ['browserify']
+    },
+
+    browserify: {
+      debug: true,
+      transform: [ 'reactify', '6to5ify' ]
+    },
+
+
+    // test results reporter to use
+    // possible values: 'dots', 'progress'
+    // available reporters: https://npmjs.org/browse/keyword/karma-reporter
+    reporters: ['progress'],
+
+
+    // web server port
+    port: 9876,
+
+
+    // enable / disable colors in the output (reporters and logs)
+    colors: true,
+
+
+    // level of logging
+    // possible values: config.LOG_DISABLE || config.LOG_ERROR || config.LOG_WARN || config.LOG_INFO || config.LOG_DEBUG
+    logLevel: config.LOG_INFO,
+
+
+    // enable / disable watching file and executing tests whenever any file changes
+    autoWatch: false,
+
+
+    // start these browsers
+    // available browser launchers: https://npmjs.org/browse/keyword/karma-launcher
+    browsers: ['Firefox'],
```

Comment:
I'm not sure how long does this take to run, but at least on travis, we should aim to support multiple targets.